__tests__/app.test.js

```diff
@@ -217,3 +217,44 @@ describe("GET /api/articles", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of comments for the given article ID with mentioned properties", () => {
+    const article_id = 3;
+    return request(app)
+      .get(`/api/articles/${article_id}/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
```

Comment:
I can't seem to find a fix for Rose's comment - but this looks like it's still missing the checking of the length, for the array, like you had with the empty one